### ISSUE TITLE
Something I have noticed.

### ISSUE THREAD

**Issue by Claycorp** [sinkillerj/ProjectE#215]
I have been watching the repo and the issues list quite extensively lately as i'm using the mod and finding bugs. 
@MozeIntel You seem to have a weird way of updating by taking some changes and bugfixes and making a "Development" build and push the whole chunk at once. The side effect of this is the issue list constantly has duplicate reports because the fixes have been around for weeks but theres no download for it.  (Example: People losing what they have learned to server crashes)

It would be great if you could push what you do more often so people that help know what's going on and that way people don't waist time on something you have done. Also whenever you push major bugfixes (Crashes fixes, Data save changes, ETC) a build would be a good idea so people using the mod don't need to continuously report the bug.

TL:DR More pushes to git to help people who make PR's. Make builds after major bug fixes are done.  
This would benefit everyone including you. You could also look into automated build systems like jenkins.
